acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -163,6 +163,10 @@ export default function init(element) {
 
   widget.querySelector('div').id = 'VERB';
   const VERB = widget.querySelector('div').textContent.trim().toLowerCase();
+  if (VERB === 'rotate-pages') {
+    const body = document.querySelector('body');
+    body.classList.add('l2-state');
```

Comment:
I like this feature-based approach better than citing the verb name directly in our CSS.  This should make it easier to migrate fill&sign styles as well.